boards/shields/x_nucleo_wb05kn1/Kconfig.defconfig

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2024 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_X_NUCLEO_WB05KN1_UART || SHIELD_X_NUCLEO_WB05KN1_SPI
+
+if BT
+
+config SPI
+	default y
+	depends on SHIELD_X_NUCLEO_WB05KN1_SPI
+
+config SPI_STM32_INTERRUPT
+	default y
+	depends on SPI
```

Comment:
Would this shield work on non-STM32 based boards?

Reply:
Yes, it works through SPI or UART. I have already tested on nRF52840-DK.